### PR TITLE
Add note to clarify this upgrading is about the OS

### PR DESCRIPTION
**Pull request by spring1843**
**1. Issue, if available:**

**2. Description of changes:**
This seems a bit vague, 'upgrade' could be interpreted in different ways including vertically scaling the nodes. This addition makes it clear that upgrading is about upgrading the operating system (i.e. the AMI) and only relevant when using AL2 and Bottlerocket.

**3. How was this change tested?**

**4. Does this change impact docs?**
- [x] Yes, PR includes docs updates
- [ ] Yes, issue opened: *link to issue*
- [ ] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/preview/concepts/_index.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ For more details on how Karpenter deletes nodes, see [Deprovisioning nodes](../t
 ### Upgrading nodes
 
 A straight-forward way to upgrade nodes is to set `ttlSecondsUntilExpired`.
-Nodes will be terminated after a set period of time and will be replaced with newer nodes.
+Nodes will be terminated after a set period of time and will be replaced with newer nodes with the most recent AMI if using AL2 or Bottlerocket.
```
Comment (bwagner5) [inline website/content/en/preview/concepts/_index.md:75]: can we say: 

"...newer nodes using the latest [EKS Optimized AMI](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-amis.html) or the AMI specified in the `$LATEST` version of your launch template."

File: website/content/en/preview/concepts/_index.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ For more details on how Karpenter deletes nodes, see [Deprovisioning nodes](../t
 ### Upgrading nodes
 
 A straight-forward way to upgrade nodes is to set `ttlSecondsUntilExpired`.
-Nodes will be terminated after a set period of time and will be replaced with newer nodes.
+Nodes will be terminated after a set period of time and will be replaced with newer using the latest [EKS Optimized AMI](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-amis.html) or the AMI specified in the `$LATEST` version of your launch template.
```
Comment (bwagner5) [inline website/content/en/preview/concepts/_index.md:75]: oops looks like `nodes` got left off after `newer`

File: website/content/en/preview/concepts/_index.md [hunk 3]
```diff
@@ -72,7 +72,7 @@ For more details on how Karpenter deletes nodes, see [Deprovisioning nodes](../t
 ### Upgrading nodes
 
 A straight-forward way to upgrade nodes is to set `ttlSecondsUntilExpired`.
-Nodes will be terminated after a set period of time and will be replaced with newer nodes.
+Nodes will be terminated after a set period of time and will be replaced with newer nodes using the latest [EKS Optimized AMI](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-amis.html) or the AMI specified in the `$LATEST` version of your launch template.
```
Comment (ellistarn) [inline website/content/en/preview/concepts/_index.md:75]: note that it's not just EKSOptimized, it's any AMI (e.g. Bottlerocket)

Comment (bwagner5) [reply website/content/en/preview/concepts/_index.md:75]: AL2, Ubuntu, Bottlerocket, and Windows are "EKS Optimized" AMI according to the link: https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-amis.html

**Pull request merged**